cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -141,24 +141,24 @@ These are the available commands:
    ``list spark``,"Lists all Spark Programs."
```

Comment:
See notes at: https://wiki.continuuity.com/display/DOCS/Building+and+Staging+CDAP+Documentation
(It's at the bottom of the page, last item.) If you need more help, just ping me.